compiler/src/iree/compiler/Codegen/SPIRV/KernelConfig.cpp

```diff
@@ -42,6 +43,12 @@ constexpr int kMaxVectorNumBits = 128;
 namespace mlir {
 namespace iree_compiler {
 
+llvm::cl::opt<std::string> clSPIRVCodegenTransformDialectFileName(
+    "iree-codegen-spirv-use-transform-dialect",
```

Comment:
We can just use `iree-spirv-` as the prefix here; no need for the additional `codegen` part.

Reply:
The cuda equivalent flag has `iree-codegen-llvmgpu...` so I think it is nice to only have to change the `llvmgpu` to `spirv` here

Reply:
we should drop the codegen part there too. :) can happen in a separate patch though. the convention is to use `iree-codgen-` prefix to mean something that is common w.r.t. all backends.

Reply:
I see, that makes sense. I'll drop the `codegen` here for now then